### PR TITLE
[MMA] Hylian Adventurer

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
Mods "Hylian Adventurer Profession for Mythical Martial Arts"

#### Purpose of change
Adds a profession to MMA.

#### Describe the solution
Make new profession. Click Submit.

#### Describe alternatives you've considered
No click submit?

#### Testing
Loaded up and walked my cloaked adventurer up to a zombie cop and proceeded to beat him to second death with my 2 by sword.

### CODE REVIEW

**Comment by Termineitor244** [comment]
Excellent! I really should add a new "Dimensional Traveler" scenario for this kind of professions haha.

And, could you add the 2-by-sword to the profession? To represent the [Forest Dweller's Sword](https://zelda.fandom.com/wiki/Forest_Dweller%27s_Sword).

**Comment by Maleclypse** [comment]
> Excellent! I really should add a new "Dimensional Traveler" scenario for this kind of professions haha.
> 
> And, could you add the 2-by-sword to the profession? To represent the [Forest Dweller's Sword](https://zelda.fandom.com/wiki/Forest_Dweller%27s_Sword).
> 
> Edit: That or a longsword to represent the [Master Sword](https://zelda.fandom.com/wiki/Master_Sword).

2bysword it is :)

File: data/mods/MMA/professions.json [hunk 1]
```diff
@@ -17,5 +17,17 @@
...
+  {
+    "type": "profession",
+    "id": "mma_hylian_adventurer",
+    "name": "Hylian Adventurer",
+    "description": "Sucked through a vortex into what appears to be a shadow realm, you will continue to fight for justice.",
+    "points": 6,
+    "traits": [ "style_mma_hylian", "ELFAEYES", "ELFA_EARS" ],
+    "skills": [ { "level": 3, "name": "cutting" }, { "level": 3, "name": "melee" }, { "level": 1, "name": "dodge" } ],
+    "items": {
+      "both": { "items": [ "combat_shirt", "cloak", "gloves_fingerless", "antarvasa", "ragpouch", "socks", "boots", "sword_wood" ] }
```
Comment (Termineitor244) [inline data/mods/MMA/professions.json:30]:
```suggestion
      "both": { "items": [ "combat_shirt", "cloak", "gloves_fingerless", "antarvasa", "ragpouch", "socks", "boots" ],
        "entries": [ { "item": "sword_wood", "custom-flags": [ "auto_wield" ] } ]
      }
```

**Review by Termineitor244 — COMMENTED** [review]
Maybe something like this would be better?

File: data/mods/MMA/professions.json (hunk 1 above)
Comment (Termineitor244) [inline data/mods/MMA/professions.json:30]:
```suggestion
    "items": {
      "both": {
        "items": [ "combat_shirt", "cloak", "gloves_fingerless", "antarvasa", "ragpouch", "socks", "boots" ],
        "entries": [ { "item": "sword_wood", "custom-flags": [ "auto_wield" ] } ]
```

Comment (Maleclypse) [reply data/mods/MMA/professions.json]: Thanks!

File: data/mods/MMA/professions.json [hunk 2]
```diff
@@ -17,5 +17,19 @@
...
+    "description": "Sucked through a vortex into what appears to be a shadow realm, you will continue to fight for justice.",
+    "points": 6,
+    "traits": [ "style_mma_hylian", "ELFAEYES", "ELFA_EARS" ],
+    "skills": [ { "level": 3, "name": "cutting" }, { "level": 3, "name": "melee" }, { "level": 1, "name": "dodge" } ],
+    "items": {
+      "both": {
+        "items": [ "combat_shirt", "cloak", "gloves_fingerless", "antarvasa", "ragpouch", "socks", "boots" ],
+        "entries": [ { "item": "sword_wood", "custom-flags": [ "auto_wield" ] } ]
+    }
   }
```
Comment (Maleclypse) [inline data/mods/MMA/professions.json:34]:
```suggestion
```

Comment (Maleclypse) [inline data/mods/MMA/professions.json:34]:
```suggestion
  }
}
```

**Comment by Maleclypse** [comment]
Note to self.  Need to find the trait name for Hylian Swordsmanship.

**Comment by Termineitor244** [comment]
> 
> 
> Note to self. Need to find the trait name for Hylian Swordsmanship.

Note to Maleclypse: It should be MMA_MARTIAL_ARTS_HYLIAN

At least according to the PR from when it was merged.

**Pull request merged**